lib/puppet/file_serving/content.rb

```diff
@@ -23,7 +23,7 @@ def self.from_binary(content)
 
   # This is no longer used, but is still called by the file server implementations when interacting
   # with their model abstraction.
-  def collect(source_permissions = nil)
+  def collect
```

Comment:
@MikaelSmith: in reference to removing the default argument here, @joshcooper mentioned he was on the fence about doing it. It'd be easy enough to remove it again (might require a spec update). @joshcooper thoughts?